app/components/deck-option.js

```diff
@@ -9,8 +9,8 @@ export default Component.extend({
   classNameBindings: [':deck-option', 'isSelected:deck-option-selected'],
   attributeBindings: ['style'],
 
-  style: computed('deck.defaultImageUrl', function() {
-    return `background-image: url("${this.get('deck.defaultImageUrl')}")`;
+  style: computed('deck.imageUrl', function() {
+    return `background-image: url("${this.get('deck.imageUrl')}")`;
```

Comment:
do you mind returning a new Ember.Handlebars.SafeString here? http://emberjs.com/deprecations/v1.x/#toc_binding-style-attributes